diff --git a/Makefile b/Makefile
index 46f11a4..066f7c0 100644
--- a/Makefile
+++ b/Makefile
@@ -8,13 +8,13 @@ endif
 
 ifdef DEBUG
 	CFLAGS = -pipe -std=c99 -Wall -Wextra -Wvla -Og -fno-pie -fno-PIE -ggdb3
-	LDFLAGS = -pipe -no-pie
+	LDFLAGS = -pipe 
 	ifneq ($(DEBUG),1)
 		CFLAGS += -DDEBUG
 	endif
 else
 	CFLAGS = -pipe -std=c99 -Wall -Wextra -Wvla -O3 $(LTOFLAGS) -fno-pie -fno-PIE -DNDEBUG
-	LDFLAGS = -pipe -no-pie -O3 $(LTOFLAGS) -s
+	LDFLAGS = -pipe -O3 $(LTOFLAGS) -s
 endif
 
 ifdef STATIC
